docs/faq/end-users.md

```diff
@@ -1,49 +1,90 @@
-# End users
+# End User FAQ
```

Comment:
Notice the `#` determines the level of headers. It is commonly set just one top header (`#`), then sub-headers `##`, ...
This headers are parsed by `docsify` to build side bar index. 

I woud suggest to have just one `#` in the whole document in order to have better looking index